__init__.py

```diff
@@ -616,26 +665,21 @@ def handle_display_manager(self, state):
             override_animations=True)
 
     def show_pairing_start(self):
-        # Make sure code stays on display
-        self.enclosure.deactivate_mouth_events()
-        self.enclosure.mouth_text(self.settings["pairing_url"] + "      ")
         self.handle_display_manager("PairingStart")
         # self.gui.show_page("pairing_start.qml", override_idle=True,
         # override_animations=True)
 
     def show_pairing(self, code):
         # self.gui.remove_page("pairing_start.qml")
-        self.enclosure.deactivate_mouth_events()
-        self.enclosure.mouth_text(code)
-        self.gui["txtcolor"] = self.settings["color"]
-        self.gui["backendurl"] = self.settings["pairing_url"]
+        # TODO - system theme
+        self.gui["txtcolor"] = self.settings.get("color") or "#FF0000"
```

Comment:
@AIIX this should be removed and use the new system themes, can you look into it?